code/game/machinery/slotmachine.dm

```diff
@@ -12,90 +12,94 @@
 	var/resultlvl = null
 
 /obj/machinery/slot_machine/attack_hand(mob/user as mob)
+	tgui_interact(user)
```

Comment:
All `ui_` procs have been replaced with `tgui_` procs for the sake of separation. This should be called from `attack_hand()` or `attack_ai`, and **must** pass `user` as an argument.